cpp/src/lib/rora_proxy_client.cc

```diff
@@ -154,53 +166,154 @@ void RoraProxy_client::_maybe_update_osd_infos(
   }
 }
 
-int RoraProxy_client::_short_path_one(const std::string & /*namespace_*/,
-                                      const ObjectSlices &object_slices,
-                                      std::shared_ptr<Manifest> mfp) {
+Location get_location(ManifestWithNamespaceId &mf, uint64_t pos, uint32_t len) {
+  int chunk_index = -1;
+  uint64_t total = 0;
+
+  {
```

Comment:
it's not needed. the part in the block was in another function earlier on, so 'historic' reasons.
it releases the iterator a bit earlier, but I can remove it if you would prefer that.